action.yml

```diff
@@ -240,27 +250,28 @@ runs:
       run: |
         npx rock build:android \
           --variant "${{ inputs.variant }}" \
+          ${{ env.BUILD_TYPE == 'aab' && '--aab' || '' }} \
           ${{ inputs.rock-build-extra-params }}
       shell: bash
       working-directory: ${{ inputs.working-directory }}
 
     - name: Find Build Artifact
       if: ${{ !env.ARTIFACT_URL }}
       run: |
-        APK_PATH=$(find $ANDROID_SOURCE_DIR/$APP_NAME/build/outputs -name '*.apk' | head -1 )
-        echo APK_PATH $APK_PATH
-        echo "ARTIFACT_PATH=$APK_PATH" >> $GITHUB_ENV
+        BINARY_PATH=$(find $ANDROID_SOURCE_DIR/$APP_NAME/build/outputs -name '*.${{ inputs.aab == 'true' && 'aab' || 'apk' }}' | head -1 )
```

Comment:
let's create BINARY_EXT variable and use that instead of `${{ inputs.aab == 'true' && 'aab' || 'apk' }}`